.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+*.sh text eol=lf
+*.bat text eol=crlf
```

Comment:
Hi, could you explain why we are setting these? Thanks, I am just trying to understand what is going on here

Reply:
It resolved the conflicts of handling "line end compile"  between different operating systems. Adding this file will solve the problem when pull from master, showing a lot of changes in local.